microservices/nris_api/backend/app/commands.py

```diff
@@ -2,26 +2,34 @@
 
```

Comment:
I see that you removed click as dependancy but are importing click on top of this file. Is python not throwing an error on that?

Reply:
Click is included with Flask now as a nested dependency. I tried to only include top level